### PR TITLE
feat(docs): document `teams` package

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: stdlib/contrib/sranka/teams/teams.flux [hunk 1]
```diff
@@ -1,18 +1,51 @@
...
 option summaryCutoff = 70
 
-// `message` sends a single message to Microsoft Teams via incoming web hook.
-// `url` - string - incoming web hook URL
-// `title` - string - Message card title.
-// `text` - string - Message card text.
-// `summary` - string - Message card summary, it can be an empty string to generate summary from text.
+// message sends a single message to a Microsoft Teams channel using an [incoming webhook].
+//
+// [incoming webhook]: https://docs.microsoft.com/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook
```
Comment (sanderson) [inline stdlib/contrib/sranka/teams/teams.flux:19]:
```suggestion
// message sends a single message to a Microsoft Teams channel using an
// [incoming webhook](https://docs.microsoft.com/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).
```

File: stdlib/contrib/sranka/teams/teams.flux [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
+//
+// ## Parameters
+//
+// - url: Incoming webhook URL.
+// - title: Message card title.
+// - text: Message card text.
+// - summary: Message card summary.
+//   Default is `""`.
+//   If no summary is provided, Flux generates the summary from the message text.
```
Comment (sanderson) [inline stdlib/contrib/sranka/teams/teams.flux:28]:
```suggestion
// - summary: Message card summary.
//   Default is `""`.
//
//   If no summary is provided, Flux generates the summary from the message text.
```

File: stdlib/contrib/sranka/teams/teams.flux [hunk 3]
```diff
@@ -1,18 +1,51 @@
...
+//     |> filter(fn: (r) => r._measurement == "statuses")
+//     |> last()
+//     |> findRecord(fn: (key) => true, idx: 0)
+//
+// teams.message(
+//   url: "https://outlook.office.com/webhook/example-webhook",
+//   title: "Disk Usage"
+//   text: "Disk usage is: *${lastReported.status}*.",
+//   summary: "Disk usage is ${lastReported.status}"
+// )
```
Comment (sanderson) [inline stdlib/contrib/sranka/teams/teams.flux:47]:
```suggestion
// import "contrib/sranka/teams"
// 
// lastReported = from(bucket: "example-bucket")
//     |> range(start: -1m)
//     |> filter(fn: (r) => r._measurement == "statuses")
//     |> last()
//     |> findRecord(fn: (key) => true, idx: 0)
// 
// teams.message(
//     url: "https://outlook.office.com/webhook/example-webhook",
//     title: "Disk Usage",
//     text: "Disk usage is: *${lastReported.status}*.",
//     summary: "Disk usage is ${lastReported.status}",
// )
```

File: stdlib/contrib/sranka/teams/teams.flux [hunk 4]
```diff
@@ -1,18 +1,51 @@
...
+// ```
```
Comment (sanderson) [inline stdlib/contrib/sranka/teams/teams.flux:48]:
```suggestion
// ```
//
// tags: single notification
//
```

File: stdlib/contrib/sranka/teams/teams.flux [hunk 5]
```diff
@@ -37,10 +70,46 @@ message = (url, title, text, summary="") => {
...
 }
 
-// `endpoint` creates the endpoint for the Microsoft Teams external service.
-// `url` - string - URL of the incoming web hook.
-// The returned factory function accepts a `mapFn` parameter.
-// The `mapFn` must return an object with `title`, `text`, and `summary`, as defined in the `message` function arguments.
+// endpoint sends a message to a Microsoft Teams channel using data from table rows.
+//
+// ## Parameters
+// - url: Incoming webhook URL.
```
Comment (sanderson) [inline stdlib/contrib/sranka/teams/teams.flux:76]:
```suggestion
// - url: Incoming webhook URL.
// - tables: Input data. Default is piped-forward data (`<-`).
```

File: stdlib/contrib/sranka/teams/teams.flux [hunk 6]
```diff
@@ -37,10 +70,46 @@ message = (url, title, text, summary="") => {
...
+//   |> range(start: -1m)
+//   |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")
+//
+// crit_statuses
+//   |> endpoint(mapFn: (r) => ({
+//       title: "Disk Usage"
+//       text: "Disk usage is: **${r.status}**.",
+//       summary: "Disk usage is ${r.status}"
+//     })
+//   )()
```
Comment (sanderson) [inline stdlib/contrib/sranka/teams/teams.flux:111]:
```suggestion
// import "contrib/sranka/teams"
//
// url = "https://outlook.office.com/webhook/example-webhook"
// endpoint = teams.endpoint(url: url)
//
// crit_statuses = from(bucket: "example-bucket")
//     |> range(start: -1m)
//     |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")
//
// crit_statuses
//     |> endpoint(
//         mapFn: (r) => ({
//             title: "Disk Usage",
//             text: "Disk usage is: **${r.status}**.",
//             summary: "Disk usage is ${r.status}",
//         }),
//     )()
```

File: stdlib/contrib/sranka/teams/teams.flux [hunk 7]
```diff
@@ -37,10 +70,46 @@ message = (url, title, text, summary="") => {
...
+// ```
```
Comment (sanderson) [inline stdlib/contrib/sranka/teams/teams.flux:112]:
```suggestion
// ```
//
// tags: notification endpoints,transformations
//
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: stdlib/contrib/sranka/teams/teams.flux [hunk 8]
```diff
@@ -37,10 +72,51 @@ message = (url, title, text, summary="") => {
...
 
-// `endpoint` creates the endpoint for the Microsoft Teams external service.
-// `url` - string - URL of the incoming web hook.
-// The returned factory function accepts a `mapFn` parameter.
-// The `mapFn` must return an object with `title`, `text`, and `summary`, as defined in the `message` function arguments.
+// endpoint sends a message to a Microsoft Teams channel using data from table rows.
+//
+// ## Parameters
+// - url: Incoming webhook URL.
+// - tables: Input data. Default is piped-forward data (`<-`).
```
Comment (sanderson) [inline stdlib/contrib/sranka/teams/teams.flux:79]:
```suggestion
```

**Review by sanderson — APPROVED** [review]
Suggestion to remove `tables` from `endpoint` and then it's gtg.

**Review by jsternberg — APPROVED** [review]

**Pull request merged**